README.md

```diff
@@ -34,41 +34,33 @@ We have a Parsons Docker container hosted on [DockerHub](https://cloud.docker.co
 
 ### Quickstart
 
+For this Quickstart, we are looking to generate a list of voters with cell phones using a [dummy data file](docs/quickstart.csv). We use the `assert` statements to verify that the data has been loaded correctly.
+
```

Comment:
Can we put a quick line in here about authentication?  We could do a short description with a link to the Github connector docs for more info.

Reply:
Or does Github require no authentication whatsoever for public repositories?

Reply:
Yeah. GitHub doesn't require auth for public repos.